types/src/jwks/rsa/mod.rs

```diff
@@ -60,6 +60,7 @@ impl RSA_JWK {
 
     // TODO(zkid): Move this to aptos-crypto so other services can use this
     pub fn to_poseidon_scalar(&self) -> Result<ark_bn254::Fr> {
+        // TODO(zkid): finalize the jwk hashing scheme.
```

Comment:
can remove, since circuit is more or less finalized

Reply:
Please do! 🙏